docs/osr-resources/InnerSource.md

```diff
@@ -59,10 +60,12 @@ Presentation by Chamindra de Silva to FINOS Members Meeting on June 15
         InnerSource project in your Financial institution. There many
         projects in finance that will always remain behind closed
         doors but that does not mean they cannot benefit from the
-        open source way of doing things. The SIG is a place you can
+        Open Source way of doing things. The SIG is a place you can
         learn from others in the finance domain on how they run
-        things. It is also closely tied to the InnerSource commons
-        organization</p>
+        such project and come up with a set of common standards 
+        and best practices. It is also closely tied to the 
+        InnerSource Commons which is a major custodian of the 
```

Comment:
Should be "InnerSource Commons organisation"

Reply:
agreed